### PR TITLE
Removed accountId parameter from requestRecentGames

### PR DESCRIPTION
**Pull request by yuntonyx**
Solves Issue #32

### CODE REVIEW

**Comment by muratsu** [comment]
fwiw this is a breaking change. can you check if more than one parameter is provided and handle that case for backwards compat?

**Comment by yuntonyx** [comment]
Oops totally forgot about that.. Working on it right now

**Comment by yuntonyx** [comment]
Not sure what the most elegant solution for this is...suggestions are welcome :c

**Comment by joshuaferrara** [comment]
I left a note on 51a35c7 

Also, can we try and keep the bracing style similar to what's being used within the [rest of the project](http://c2.com/cgi/wiki?OneTrueBraceStyle)? Other than that, while it isn't the most elegant, it looks good to me.

**Comment by yuntonyx** [comment]
I have made the appropriate changes!

**Comment by muratsu** [comment]
:+1: lgtm

File: handlers/match.js [hunk 1]
```diff
@@ -103,14 +103,26 @@ CSGO.CSGOClient.prototype.requestGame = function(matchid, outcome, token, callba
...
   if (!this._gcReady) {
     if (this.debug) {
       util.log("GC not ready");
     }
     return null;
   }
+  
+  var accid = this.ToAccountID(this._user._client.steamID);
+  var callback;
+  if(arguments.length >= 2) {
```
Comment (andrewda) [inline handlers/match.js]: Just to keep everything nice and tidy, you'll probably want to follow the same if/else style and spacing that the rest of the code uses (i.e.

```
if (foo) {
  bar();
} else {
  foobar();
}
```

Besides that, looks good!

**Comment by yuntonyx** [comment]
This should do hopefully. I am surprised that it took this many commits for this little change :3

**Comment by joshuaferrara** [comment]
Fixes #33

**Pull request merged**